From 90e1fd964b278e704d52408570da0ebb55bac115 Mon Sep 17 00:00:00 2001
From: Jaroslav Kysela <perex@perex.cz>
Date: Thu, 27 Jan 2022 13:15:46 +0100
Subject: [PATCH 09/39] speaker-test: remove sample_map.csv

The CSV file is not used. The .wav file names are fixed
in the source code, but the directory may be specified
using --wavdir. Remove this file until there's a demand
for the more precise .wav file mapping.

Fixes: https://github.com/alsa-project/alsa-utils/issues/133
Signed-off-by: Jaroslav Kysela <perex@perex.cz>
---
 speaker-test/samples/sample_map.csv | 2 --
 1 file changed, 2 deletions(-)
 delete mode 100644 speaker-test/samples/sample_map.csv

diff --git a/speaker-test/samples/sample_map.csv b/speaker-test/samples/sample_map.csv
deleted file mode 100644
index 20d8ed2..0000000
--- a/speaker-test/samples/sample_map.csv
+++ /dev/null
@@ -1,2 +0,0 @@
-0, "Front Left", "/usr/share/alsa/samples/Front_Left.wav"
-1, "Front Right", "/usr/share/alsa/samples/Front_Right.wav"
-- 
2.36.0

